src/doctests.jl

```diff
@@ -14,7 +14,7 @@ struct DocTestContext
     file::String
     doc::Documenter.Document
     meta::Dict{Symbol, Any}
-    DocTestContext(file::String, doc::Documenter.Document) = new(file, doc, Dict())
+    DocTestContext(file::String, doc::Documenter.Document) = new(file, doc, copy(doc.user.default_meta))
```

Comment:
Perhaps this should only copy `DocTestSetup` and `DocTestTeardown` if present?

Reply:
It looks like when we parse the at-meta blocks when doctesting, we put everything into the `.meta` dict. So I think it's best to stay consistent here (even though we don't use the other values).